Scenes/Player/PlayerState.cs

```diff
@@ -1,17 +1,16 @@
 using System;
 using Godot;
+using YamlDotNet.Serialization;
 namespace MVM23;
 
 public interface IPlayerState {
```

Comment:
If `IPlayerState` doesn't have any other implementers, we should just delete it and use `PlayerState` instead, since it has an identical copy of `HandleInput` (`interface` methods always being `abstract`).